clients/python/src/model_registry/_client.py

```diff
@@ -573,6 +570,9 @@ def save_to_s3(
         access_key_id: str | None = None,
         secret_access_key: str | None = None,
         region: str | None = None,
+        multipart_threshold: int = 1024 * 1024,
+        multipart_chunksize: int = 1024 * 1024,
+        max_pool_connections: int = 10,
```

Comment:
great extension to the capability thanks @syntaxsdev , which is also covered by e2e test below (and yes we can always expand for stress testing)